src/app/views/sidebar/Sidebar.tsx

```diff
@@ -5,14 +5,17 @@ import { telemetry } from '../../../telemetry';
 import { translateMessage } from '../../utils/translate-messages';
 import History from './history/History';
 import SampleQueries from './sample-queries/SampleQueries';
-
+import { ResourceExplorer } from './resource-explorer';
 export const Sidebar = () => {
   return (
     <div>
       <Pivot onLinkClick={onPivotItemClick} overflowBehavior='menu'>
         <PivotItem headerText={translateMessage('Sample Queries')} itemIcon='Rocket' itemKey='sample-queries'>
           <SampleQueries />
         </PivotItem>
+        <PivotItem headerText={translateMessage('Resources')} itemIcon='ExploreData' itemKey='resources'>
```

Comment:
The `Resources` tab should remain hidden for now.

Reply:
Nice catch.
I left it here for testing. I'll remove it when merging to dev